PWGHF/HFC/TableProducer/correlatorDsHadrons.cxx

```diff
@@ -62,7 +63,8 @@ std::vector<double> multBinsMcGen{VARIABLE_WIDTH, 0., 20., 50.0, 500.}; // In MC
 using BinningType = ColumnBinningPolicy<aod::collision::PosZ, aod::mult::MultFV0M<aod::mult::MultFV0A, aod::mult::MultFV0C>>;
 BinningType corrBinning{{zBins, multBins}, true};
 
-struct HfDsSelectionCollision {
+/// Code to select collisions with at least one Ds meson
+struct HfSelCollisionDs {
```

Comment:
This is an auxiliary task within a workflow. Please name it according to the HF guidelines.

Reply:
In the documentation I've found that I've to use MainTaskName<Suffix>, so now I've called it in this way: `HfCorrelatorDsHadronsSelCollision`. Is it correct to do like this? Because I don't understand what Suffix properly means

Reply:
Yes, that is fine.